modules/serverless-channel-default.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-channel-default_{context}"]
+= Configuring the default channel type
+
+The `default-ch-webhook` ConfigMap can be used to specify the default channel implementation for the cluster and the namespace.
+
+You can make changes to the `knative-eventing` namespace ConfigMaps, including the `default-ch-webhook` ConfigMap, by using the {ServerlessOperatorName} to propagate changes.
+To do this, you must modify the `KnativeEventing` custom resource.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {product-title}.
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` custom resource to add configuration details for the `default-ch-webhook` ConfigMap:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  config: <1>
+    default-ch-webhook: <2>
+      default-ch-config: |
+        clusterDefault: <3>
+          apiVersion: messaging.knative.dev/v1beta1
+          kind: InMemoryChannel
+          spec:
+            numPartitions: 10
+            replicationFactor: 1
```

Comment:
the IMC does not have these fields, they are for Kafka.

In upstream I used these for the IMC config:
```
...
            spec:
              delivery:
                backoffDelay: PT0.5S
                backoffPolicy: exponential
                retry: 5
```